sdk/eventhub/event-hubs/README.md

```diff
@@ -250,4 +250,8 @@ Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tr
 directory for detailed examples on how to use this library to send and receive events to/from
 [Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about).
 
+## Contributing
```

Comment:
We should try to be consistent across all our readme files and call the heading "Contributing" everywhere and not "Developer Contributing Guide" in some places. For cases where you are linking from a specific contributing.md to the general on you can just add that as a pointer at the top of the specific contributing.md without a need for a header.

Reply:
We'll be using "Contributing" section everywhere. Updating all the PRs with the same.